ts/components/session/LeftPaneContactSection.tsx

```diff
@@ -321,9 +321,12 @@ export class LeftPaneContactSection extends React.Component<Props, State> {
 
   private renderList() {
     const { sentFriendsRequest } = this.props;
+
     const friends = window.getFriendsFromContacts(this.props.friends);
     const length = Number(sentFriendsRequest.length) + Number(friends.length);
-    const combined = [...sentFriendsRequest, ...friends];
+
+    // Prevent where friends and send FR showing two entries
+    const combined = [...new Set([...sentFriendsRequest, ...friends])];
```

Comment:
not sure about how that can happen

Reply:
Please see the .md

![image-20200504133017652](https://user-images.githubusercontent.com/58160433/81768115-8cf9e900-951d-11ea-9783-bfe237be88d4.png)
![image-20200512075350722](https://user-images.githubusercontent.com/58160433/81768126-971be780-951d-11ea-95d8-d8966f827f3d.png)